Networking/Networking/Mapper/WCAnalyticsCustomerMapper.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+/// Mapper: WCAnalyticsCustomer
+///
+struct WCAnalyticsCustomerMapper: Mapper {
+    /// We're injecting this field by copying it in after parsing responses, because `siteID` is not returned in any of the Customer endpoints.
+    ///
+    let siteID: Int64
+
+    /// (Attempts) to convert a dictionary into a `[WCAnalyticsCustomer]` entity
+    ///
+    func map(response: Data) throws -> [WCAnalyticsCustomer] {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(DateFormatter.Defaults.dateTimeFormatter)
+        decoder.userInfo = [.siteID: siteID]
+        let customers = try decoder.decode([WCAnalyticsCustomer].self, from: response)
+        return customers
+    }
+
+    /// (Attempts) to convert a dictionary into a `WCAnalyticsCustomer` entity
+    ///
+    func mapUniqueCustomer(response: Data, searchValue: String? = "") throws -> WCAnalyticsCustomer? {
+        if searchValue == "" {
+            return WCAnalyticsCustomer(userID: 0, name: "")
+        }
```

Comment:
This... seems strange. Why not return nil, which you'd get for free by deleting this code and leaving it to the filter to deal with it? If you return this, it'll go into storage or you'll have to code around it in the rest of the app.

Reply:
You're totally right, I'm actually not sure why I did that. Fixed on 527f5f9319a299ce5b8f215d56a5c44a6cac991a (but later on, then has been entirely removed on def5158743fcae7d44aeef8c931e23a788d77860 😬 )